File: LeetCode/LeetCode26.java
class LeetCode26{
    public static void main(String[] args) {
        int[] nums = {0,0,1,1,1,2,2,3,3,4};
        int s = new LeetCode26().removeDuplicates(nums);
        for (int i = 0; i < s; i++) {
            System.out.print(nums[i]+" ");
        }
    }
    public int removeDuplicates(int[] nums) {
        int j=1,i=0;
        for(i = 0; i < nums.length-1; i++){    
            if(nums[i]==nums[i+1]){
                nums[j++]=nums[i+1];
            }
        }
        return j;
    }
}

File: LeetCode/LeetCode26.py
from pip import List


class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        s = set(nums)
        k = len(s)
        numsL = list(s)
        numsL.sort()
        i=0
        for j in numsL:
            nums[i] = j
            i+=1
        return k

nums = [0,0,1,1,1,2,2,3,3,4]
s = Solution();
k = s.removeDuplicates(nums)
print(nums[:k+1])